File: POTW/week-7/CS_Solution7.java
/**
 * Krish Shah
 * Determine if a given year is a leap year.
 */

package potw;

import java.util.Scanner;

public class CS_Solution6 {

	public static boolean isLeapYear(int year) {
		boolean isLeapYear = false;
		if (year % 4 == 0 && year % 100 != 0) {
			isLeapYear = true;
		}
		if (year % 400 == 0) {
			isLeapYear = true;
		}
		return isLeapYear;
	}

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		int year = scanner.nextInt();
		System.out.println(isLeapYear(year));
	}

}

File: POTW/week-7/CS-Solution7.py
# Krish Shah
# Determine if a given year is a leap year.

year = int(input())

is_leap_year = False

if year % 4 == 0 and not year % 100 == 0:
    is_leap_year = True
elif year % 400 == 0:
    is_leap_year = True

if is_leap_year:
    print("yes")
else:
    print("no")
